src/app/routes/welcome/welcome.component.ts

```diff
@@ -52,4 +53,10 @@ export class WelcomeComponent implements OnInit {
   private login(provider: WalletProvider) {
     this.store.dispatch(AuthActions.welcomeLogin({provider, returnUrl: this._returnUrl}));
   }
+
+  private tryToLoginWithPrivateKey() {
+    if (window.localStorage.getItem('PrivateKey')) {
+      this.login(WalletProvider.PrivateKey);
+    }
```

Comment:
What do you think of adding this log?
```suggestion
    if (window.localStorage.getItem('PrivateKey')) {
      console.log('found privateKey. Using to login');
      this.login(WalletProvider.PrivateKey);
    }
```